resources/js/modules/show-hide/index.js

```diff
@@ -1,32 +1,49 @@
-// import $$ from '@utilities/selectors';
-
-// const ShowHide = function ShowHide() {
-//   let content,
-//       chevron;
-
-//   $$.toggleShowHide.forEach(btn => {
-//     btn.addEventListener('click', function(e) {
-//       content = this.nextElementSibling;
-//       content.maxHeight = content.scrollHeight;
-
-//       for ( let i = 0; i < this.childNodes.length; i++ ) {
-//         if (this.childNodes[i].classList.contains('chevron')) {
-//           chevron = this.childNodes[i];
-//         }
-//       }
+import $$ from '@utilities/selectors';
+
+function loopThroughChildren(parent, className) {
+  if ( !parent || !className || !parent.childNodes.length) {
+    console.log("Please check if all correct arguments are provided for loopThroughChildren() function!");
+    return;
+  }
+  for (let i = 0; i < parent.childNodes.length; i++ ) {
```

Comment:
could use a .map() here instead of for loop, they're a bit cleaner

Reply:
Would it be ok if instead map there would be a forEach loop? Because there is no need to create a new array, since the values are not modified